hazelcast/src/test/java/com/hazelcast/map/impl/querycache/QueryCacheCoalescingTest.java

```diff
@@ -32,6 +33,17 @@
     @SuppressWarnings("unchecked")
     private static final Predicate<Integer, Integer> TRUE_PREDICATE = TruePredicate.INSTANCE;
 
+    @Before
```

Comment:
The `CoalescingPublisherAccumulator` has two `logger.isFinestEnabled()` blocks which are pushing the code coverage below the threshold. This should enable those code paths.